runner_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
by convention, test files should start with “test_”. this also enables test runners such as py.test to find them by default. which means you …